<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaseSpaceDialog</class>
 <widget class="QDialog" name="CaseSpaceDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Case Space</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>792</x>
     <y>657</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="queryCaseLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>20</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Query Case:</string>
   </property>
  </widget>
  <widget class="QLabel" name="queryCaseNameLabel">
   <property name="geometry">
    <rect>
     <x>95</x>
     <y>20</y>
     <width>465</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;not selected&gt;</string>
   </property>
  </widget>
  <widget class="QVTKWidget" name="caseSpaceViewWidget" native="true">
   <property name="geometry">
    <rect>
     <x>27</x>
     <y>71</y>
     <width>920</width>
     <height>573</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>920</width>
     <height>569</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>656</y>
     <width>166</width>
     <height>25</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="compareCasesPushButton">
      <property name="text">
       <string>Compare Cases</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="testPushButton">
      <property name="text">
       <string>Test</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="matchCaseLabel">
   <property name="geometry">
    <rect>
     <x>28</x>
     <y>41</y>
     <width>60</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Match Case:</string>
   </property>
  </widget>
  <widget class="QLabel" name="matchCaseNameLabel">
   <property name="geometry">
    <rect>
     <x>94</x>
     <y>41</y>
     <width>592</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;not selected&gt;</string>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>qvtkwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CaseSpaceDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CaseSpaceDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
